src/cmake/cuda_macros.cmake

```diff
@@ -26,8 +28,9 @@ function ( NVCC_COMPILE cuda_src ptx_generated extra_nvcc_args )
             ${LLVM_COMPILE_FLAGS}
             -DOSL_USE_FAST_MATH=1
             -m64 -arch ${CUDA_TARGET_ARCH} -ptx
-            --std=c++11 -dc -O3 --use_fast_math --expt-relaxed-constexpr
+            --std=c++14 -dc -O3 --use_fast_math --expt-relaxed-constexpr
```

Comment:
Should this instead be the same version that is being used for the rest of the OSL build?

Or are you saying that from here on, the Cuda portions *must* be precisely C++14? Or is the whole system now C++14 minimum if one is building the OptiX support?

Reply:
Only the Cuda portions require C++14 for the compile-time evaluation of the string hash.  Technically, it only applies if we're compiling for OptiX7, so I can look into "lowering" the C++ requirements to C++11 for OptiX6.

Reply:
No, I think C++14 mode for Cuda is fine, I don't see how that can hurt us, even on OptiX6. And we'll probably be bumping the floor to C++14 in the coming months, as C++11 is being more and more untenable over time (C++11 users already can't build against LLVM 10 or 11).